com.orama_interactive.Pixelorama.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: com.orama_interactive.Pixelorama
+base: org.godotengine.godot.BaseApp
+base-version: '3.1'
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+default-branch: stable
+sdk: org.freedesktop.Sdk
+command: pixelorama
+rename-icon: icon
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=host
```

Comment:
Just use home filesystem access here, these permissions can be easily changed later. 
Other than this, it should be ready

Reply:
That makes sense, my only concern is that it removes the ability to edit files present on removable media (if I read the docs correctly). It's probably not a huge issue for most people, but people may wonder why they can't browse removable media.